packages/contracts-periphery/contracts/universal/op-nft/AttestationStation.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {
+    OwnableUpgradeable
+} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+contract AttestationStation is Initializable, OwnableUpgradeable, Semver {
+    struct AttestationData {
+        address about;
+        bytes32 key;
+        bytes val;
+    }
+
+    mapping(address => mapping(address => mapping(bytes32 => bytes))) public attestations;
+
+    event AttestationCreated(
+        address indexed creator,
+        address indexed about,
+        bytes32 indexed key,
+        bytes val
+    );
+
+    constructor() Semver(0, 0, 1) {}
+
+    /**
+     * @notice  Initialize the Optimist contract.
+     * @dev     Initializes the AttestationStation contract with msg.sender as owner
+     */
+    function initialize() public initializer {
+        __Ownable_init();
```

Comment:
I believe we should transfer ownership here to the desired owner so that the `msg.sender` is not the owner

Reply:
ie `initialize(address _owner)`

Reply:
ah makes sense

Reply:
See this comment before updating: https://github.com/ethereum-optimism/optimism/pull/4361#discussion_r1044821903

Reply:
got it.  So the deployment script will be what puts it behind a proxy